app/assets/javascripts/controllers.js

```diff
@@ -17,6 +17,27 @@ Prometheus.Graph = {
     's': 1
   },
 
+  durationToString: function(seconds) {
+    // TODO(stuart): this is always returning seconds
+    // look to round to a certain amount based on the units
+    // in the graph? some level of precision lost.
```

Comment:
This is up to debate. Currently, I have yet to see a range defined by dragging a selection on the graph return anything other than seconds. I'm all for removing this function and always returning seconds above (app/assets/javascripts/angular/resources/shared_graph_behavior.js:86). Up to you guys.